Corrfunc/theory/DD.py

```diff
@@ -84,9 +84,12 @@ def DD(autocorr, nthreads, binfile, X1, Y1, Z1, weights1=None, periodic=True,
     boxsize: double
```

Comment:
Do you want to add an explicit "required"?

Reply:
How about "required if periodic=True"?  It's technically not required otherwise.